doc/manual.asciidoc

```diff
@@ -204,7 +204,9 @@ Several placeholders are available:
 `%o`:: Overall rate of finished edges per second
 `%c`:: Current rate of finished edges per second (average over builds
 specified by `-j` or its default)
-`%e`:: Elapsed time in seconds.  _(Available since Ninja 1.2.)_
+`%e`:: Elapsed time in hh:mm:ss format.  _(Available since Ninja 1.2.)_
```

Comment:
I don't think changing this is nice. Can a new specifier be used instead? Also, it seems the test suite actually gets `h:mm:ss` for less-than-10-hour estimates?

Reply:
Thanks for taking a look!
I've squatted on `%e` because printing it in seconds seems rather pointless to me
yet the abbrev seems appropriate.
Are there stability guarantees for these placeholders?

Reply:
I don't know about stability guarantees, but there are definitely tools that hurk and die if a non-default `NINJA_STATUS` is used because they try to parse it for their own progress meters in the IDE UI or whatever (Qt Creator is/was one? Maybe they've gone to forcing their own `NINJA_STATUS` instead these days).

Reply:
And I do have a concern about always including the hours is that it takes up space when ~most builds are probably well under an hour (especially incremental which is where I'd expect a human to be waiting for things and watching the `%e` anyways).

Reply:
I do believe both of these concerns are now addressed :)